ivy_tests/test_ivy/test_frontends/test_torch/test_loss_functions.py

```diff
@@ -4,57 +4,84 @@
 
 # local
 import ivy_tests.test_ivy.helpers as helpers
+from ivy_tests.test_ivy.helpers import handle_cmd_line_args
 import ivy.functional.backends.numpy as ivy_np
 import ivy.functional.backends.torch as ivy_torch
 
 
 # cross_entropy
+@handle_cmd_line_args
 @given(
     dtype_and_input=helpers.dtype_and_values(
         available_dtypes=tuple(
             set(ivy_np.valid_float_dtypes).intersection(
```

Comment:
We can probably accept ints for this one

Reply:
No, because the `torch.nn.functional.cross_entropy` is a log softmax and only accepts floats and doubles as inputs.

Reply:
Ok sounds good